### PR TITLE
REDSHOP-6068: Order Detail - Special Discount show dialog when user input correct

### PR DESCRIPTION
**Pull request by hoangvan149dhv**

### CODE REVIEW

File: component/admin/models/order_detail.php [hunk 1]
```diff
@@ -875,7 +875,15 @@ public function special_discount($data, $chk = false)
         $orderData->special_discount        = $specialDiscount;
         $orderData->special_discount_amount = $discountPrice;
 
-        $orderData->order_total = $orderSubTotal + $orderData->order_shipping - $discountPrice - $orderData->order_discount;
+        if($discountPrice >= $orderSubTotal){
+            $orderData->special_discount  = 100 ;var_dump($orderData->special_discount );
```
Comment (anhredweb) [inline component/admin/models/order_detail.php]: @hoangvan149dhv `DEBUG` code not allow exist in PR

Comment (anhredweb) [inline component/admin/models/order_detail.php]: @hoangvan149dhv Why you assign hard code value `100` for special discount

File: component/admin/views/order_detail/tmpl/default.php [hunk 2]
```diff
@@ -1596,10 +1596,10 @@ function validateDiscount(form) {
...
         var totalDiscount = discount + specialDiscount;
 
-        if (subTotal <= totalDiscount) {
-            alert('<?php echo JText::_("COM_REDSHOP_ORDER_DISCOUNT_INVALID") ?>');
-            return false;
-        }
+        // if (subTotal <= totalDiscount) {
+        //     alert('<?php echo JText::_("COM_REDSHOP_ORDER_DISCOUNT_INVALID") ?>');
+        //     return false;
+        // }
```
Comment (anhredweb) [inline component/admin/views/order_detail/tmpl/default.php]: @hoangvan149dhv Commented code not allowed to exist

**Comment by anhredweb** [comment]
@hoangvan149dhv Please add label and milestone for PR, ask @NguyenBao10 for help if you dont know

**Comment by NguyenBao10** [comment]
dupplicated https://github.com/redCOMPONENT-COM/redSHOP/pull/5487

**Pull request closed without merging**